Tests/Pcap++Test/Tests/LiveDeviceTests.cpp

```diff
@@ -502,18 +502,22 @@ PTF_TEST_CASE(TestPcapLiveDeviceSpecialCfg)
 	PTF_ASSERT_GREATER_THAN(packetCount, 0);
 
 #ifdef HAS_SET_DIRECTION_ENABLED
-	// create a non-default configuration with only cpturing incoming packets and open the device again
-	pcpp::PcapLiveDevice::DeviceConfiguration devConfigWithDirection(pcpp::PcapLiveDevice::Promiscuous, 10, 2000000, pcpp::PcapLiveDevice::PCPP_IN);
+	packetCount = 0;
 
-	liveDev->open(devConfigWithDirection);
+	// create a non-default configuration with only capturing incoming packets and open the device again
+	pcpp::PcapLiveDevice::DeviceConfiguration devConfigWithDirection(pcpp::PcapLiveDevice::Promiscuous, 10, 2000000, pcpp::PcapLiveDevice::PCPP_OUT);
```

Comment:
I think changing from `pcpp::PcapLiveDevice::PCPP_IN` to `pcpp::PcapLiveDevice::PCPP_OUT` solved it, probably because we use `tcpreplay` in the CI so all packets seem to be outgoing instead of incoming